test/e2e/network_test.go

```diff
@@ -144,6 +145,35 @@ func testEastWestNetworkingAcrossWindowsNodes(t *testing.T) {
 	}
 }
 
+// testEastWestNetworkingMetrics deploys Linux pod and tests that it can communicate with Windows node's metrics port
+func testEastWestNetworkingMetrics(t *testing.T) {
+	testCtx, err := NewTestContext(t)
+	require.NoError(t, err)
+
+	// Need at least one Windows node to run these tests, throwing error if this condition is not met
+	require.GreaterOrEqualf(t, len(gc.nodes), 1, "insufficient number of Windows nodes to run tests across"+
+		" nodes, Minimum node count: 1, Current node count: %d", len(gc.nodes))
+
+	for _, winNode := range gc.nodes {
+		// Get the node internal IP so we can curl it
+		winNodeInternalIP := winNode.Status.Addresses[0].Address
```

Comment:
Is the 0th address always going to be the internal IP?

Reply:
As of now (with Windows nodes no longer having external IPs), internal IP will be 0th. I have added some logic to make it more extendable however. Thanks.